exporter.go

```diff
@@ -16,5 +18,8 @@ func NewBR(c *Cluster) Exporter {
 
 func (l *BR) Export() error {
 	// todo @zeyuan
-	return nil
+	cmd := exec.Command(l.c.brPath, " backup full", "--pd", l.c.pd,
```

Comment:
I think we can provide an interface to use the commands, you can refer to the previous go-ycsb error handle